test/CRM/ElifeToken/Token/ArticlesLast7DaysTest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+require_once __DIR__.'/../../../../CRM/ElifeToken/Token/ArticlesLast7Days.php';
+
+class CRM_ElifeToken_Token_ArticlesLast7DaysTest extends PHPUnit_Framework_TestCase {
+  public static function possibleTypes() {
+    return [
+        // not sure these are the right inputs, but they would 
+        // be a good naming convention
```

Comment:
to be checked, if changed we'll map the values to these two filters

Reply:
giorgio, i was using 'POA', 'VOR', and 'Magazine' since we need to implement all three from those results.

I was implementing this just now (didn't realise that you were also). I've also implemented the filter for magazine. But happy to use your implementation if you prefer.